dev/cli

```diff
@@ -31,32 +31,48 @@ GLOBAL OPTIONS
 COMMANDS
     --authn-oidc    Enables OIDC features
     --authn-azure   Enables Azure features
-    --authn-gce     Enables GCE features
+    --authn-gce [GCE_INSTANCE_NAME] Enables GCE features passing in the Google Compute Engine VM name
 EOF
 exit
 }
 
 function enable_gce() {
+  local gce_instance_name="$1"
+  local gce_zone="$2"
+
+  if [ -z "$gce_instance_name" ]; then
+    local prgname=$(basename "$0")
+    echo "Missing GCE instance name. Usage ./$prgname exec --authn-gce [GCE_INSTANCE_NAME] --zone=[GCE_ZONE], \
+(--zone is optional: default zone is 'us-central1-a')"
+    exit 1
+  fi
+
+  echo "Obtain GCE ID tokens from GCE instance: '$gce_instance_name' and persist tokens"
+  ../ci/authn-gce/get_gce_id_tokens.sh "$gce_instance_name" "$gce_zone"
+
   gce_token_file="../ci/authn-gce/tokens/gce_token_valid"
   if ! [ -f "$gce_token_file" ]; then
     echo "GCE token file $gce_token_file does not exist."
     exit 1
   fi
-
   decoded_gce_token_payload=$(_get_gce_token_payload "$gce_token_file")
 
   echo "Setting GCE details as env variables"
   GCE_PROJECT_ID="$(echo "$decoded_gce_token_payload" | jq -r '.google.compute_engine.project_id')"
   GCE_INSTANCE_NAME="$(echo "$decoded_gce_token_payload" | jq -r '.google.compute_engine.instance_name')"
   GCE_SERVICE_ACCOUNT_ID="$(echo "$decoded_gce_token_payload" | jq -r '.sub')"
   GCE_SERVICE_ACCOUNT_EMAIL="$(echo "$decoded_gce_token_payload" | jq -r '.email')"
-
+  if [ -z "$GCE_PROJECT_ID" ]; then
```

Comment:
please add a comment that it is enough to check only one or alternatively check all of them